### ISSUE TITLE
Translate Tavern (Place/Tavern.txt)

### ISSUE THREAD

**Issue by Giratena** [Giratena/Succubus-Duel-English-Translation#144]

**Comment by Giratena** [comment]
From @HotHeadEmperor:

> I ran it through a translator a while back and it seemed like it was just the accessory girl scene from the village with the bandits, which is already translated in the other location file

It's possible stuff in this file may not need to be translated as they could be remnants of old stuff.

**Comment by WisDZ** [comment]
Yep! Analyzed it and this all appears to be remnants of older stuff.

There's a small section where Milene speaks two lines as if you're entering her tavern, but the remaining dialogue is all a subset of the dialogue from the village before Necro (which was moved to a different file before being completed - WestTown.txt)

Example excerpt with differences bolded - Village Boy A

Tavern.txt
> //情報収集
@村の男A
    **%男,=,%%80001
    話者,%男,true,**
    セリフ,お、あんたが依頼を受けてくれたカードマスターか？\n頼むぜ、盗賊たちのせいで、この村もだいぶやべえんだ。
    セリフ,金目のものはもちろん、食い物も奪っていっちまう。\n普通の盗賊なら俺たちで追い返せないこともねえけど、\n相手には強いカードマスターが味方してるんだよ。
    セリフ,盗賊退治に行った村の腕自慢たちが、みんなカードマスターに負けて、\n足腰立たない骨抜きにされて帰ってきたって話だぜ。
    セリフ,だからあんたが頼りなんだ。\n悪いカードマスターに負けないでくれよ。
    $$西の村情報A,=,1

WestTown.txt
> //情報収集
@村の男A
    **話者,%%男NPC,true,**
    セリフ,お、あんたが依頼を受けてくれたカードマスターか？\n頼むぜ、盗賊たちのせいで、この村もだいぶやべえんだ。
    セリフ,金目のものはもちろん、食い物も奪っていっちまう。\n普通の盗賊なら俺たちで追い返せないこともねえけど、\n相手には強いカードマスターが味方してるんだよ。
    セリフ,盗賊退治に行った村の腕自慢たちが、みんなカードマスターに負けて、\n足腰立たない骨抜きにされて帰ってきたって話だぜ。
    セリフ,だからあんたが頼りなんだ。\n悪いカードマスターに負けないでくれよ。
    $$西の村情報A,=,1

**Comment by WisDZ** [comment]
OK to close, or I can add text to the file indicating that it is text that should not appear in a playthrough of the normal game (as below).

> アクセサリー売りの女,TEXT ERROR: Place-Tavern-Line6: If you see this; please report in the Bugs channel in Discord.

**Comment by Giratena** [comment]
Closing this for now as it seems to be OK to leave ^^
